.github/SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
```

Comment:
Why are we putting SECURITY.md into `.github/`? Scikit-learn has it in their root directory.

Reply:
Will move it to root